docs/source/how-to-guides/debug-iso15118.rst

```diff
@@ -0,0 +1,138 @@
+.. _htg_debug_iso15118:
+
+#################
+Debug ISO 15118
+#################
+
+This how-to-guide will show you how to debug ISO15118 communication
+of EVerest using Wireshark.
+
+.. _htg_wireshark:
+
+Wireshark plugin
+=================
+
+Using the Wireshark plugin from *dspace*, it is possilbe to decode the
+EXI messages sent over the interface of the PLC modem. The plugin uses
+the same EXI decoder as EVerest.
+
+It does even decrypt TLS-encrypted communication as EVerest sends the
+keys to decrypt the packages as UDP broadcast, so the plugin can make
+use of it. It is possilbe to live-view traffic between EV and EVSE and
+to decrypt captured traffic after the fact.
+
+Install
+-------
+
+Follow the steps in this
+`README <https://github.com/dspace-group/dsV2Gshark?tab=readme-ov-file#requirements>`_
+to install the plugin:
+
+.. tip::
+
+   Make sure to copy the **whole** content of the zipped archive to the
+   corresponding folder!
+
+.. _remotelivecapture:
+
+Remote live capture
+--------------------
+
+In order to view live traffic, an *ssh* connection between the system(s)
+running Wireshark and the target is required.
+
+Making use of a VPN connection enables remote live capture from around
+the globe, e.g. developers in the backoffice observing live traffic on
+chargers during testivals.
+
+To start a remote live capture session in Wireshark, click on the small
+gear symbol next to:
+
+.. image:: images/sshdumpsetupsymbol.png
```

Comment:
This image is not centered like the other images:
 
<img width="1506" height="1672" alt="image" src="https://github.com/user-attachments/assets/b69ec8c6-4d3f-4893-88d9-07a36e397e92" />

Reply:
fixed it :)